source/tutorial/ex4.rst

```diff
@@ -0,0 +1,50 @@
+.. _Exercise 4:
+
+Exercise 4 \\- \ Boot to Linux
+---------------
+
+.. note::
+  **In this exercise, we'll learn how to boot to Linux.**
+
+
+You can execute |SPN| with the following steps:
+
+* Wrking on Command Prompt Shell window
+
+
+* Rebuild SlimBoot by type-in cmd::
+
+    python BuildLoader.py build qemu
+
+* Completion: you will see ``Done [qemu]`` on the screen after compile completed
+
+* Execute bootloader on QEMU by type-in cmd:
+
+ - Windows::
+ 
+    "C:\Program Files\qemu\qemu-system-x86_64.exe" -m 256M -machine q35 -serial stdio -pflash Outputs\qemu\SlimBootloader.bin -drive id=mydisk,if=none,file=..\Misc\QemuImg\QemuSata.img,format=raw -device ide-hd,drive=mydisk -boot order=d
```

Comment:
Where does the user acquire this disk image from?

Reply:
We'll add methods for how to build and config image